openfeature/transaction_context/__init__.py

```diff
@@ -0,0 +1,15 @@
+from openfeature.transaction_context.context_var_transaction_context_propagator import (
+    ContextVarsTransactionContextPropagator,
+)
+from openfeature.transaction_context.no_op_transaction_context_propagator import (
+    NoOpTransactionContextPropagator,
+)
+from openfeature.transaction_context.transaction_context_propagator import (
+    TransactionContextPropagator,
+)
+
+__all__ = [
+    "TransactionContextPropagator",
+    "NoOpTransactionContextPropagator",
```

Comment:
to keep it aligned with the other `NoOp` classes I would prefer not to expose it here and when needed just import it via the `no_op_transaction_...` file directly.